cmd/kubeadm/app/master/apiclient.go

```diff
@@ -68,9 +68,12 @@ func CreateClientAndWaitForAPI(file string) (*clientset.Clientset, error) {
 	fmt.Println("[apiclient] Created API client, waiting for the control plane to become ready")
 
 	start := time.Now()
-	wait.PollInfinite(apiCallRetryInterval, func() (bool, error) {
+	err = wait.PollInfinite(apiCallRetryInterval, func() (bool, error) {
 		cs, err := client.ComponentStatuses().List(v1.ListOptions{})
```

Comment:
also, as a TODO, the componentstatuses API assumes things about the cluster topology we should not assume (control plane healthz available over localhost unsecured ports, etcd is unsecured, etc). I would probably switch this to check the API /healthz instead